### PR TITLE
remove debug images name and point to ocp images

### PR DESCRIPTION
**Pull request by msherif1234**
Signed-off-by: Mohamed Mahmoud <mmahmoud@redhat.com>

### CODE REVIEW

File: bundle/manifests/ingress-node-firewall.clusterserviceversion.yaml [hunk 1]
```diff
@@ -239,14 +239,14 @@ spec:
...
-                  value: quay.io/mmahmoud/ingress-node-firewall-daemon:latest
+                  value: quay.io/openshift/origin-ingress-node-firewall-daemon:latest
                 - name: DAEMONSET_NAMESPACE
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
                 - name: KUBE_RBAC_PROXY_IMAGE
                   value: quay.io/openshift/origin-kube-rbac-proxy:latest
-                image: quay.io/mmahmoud/controller:latest
+                image: quay.io/mmahmoud/ingress-node-firewall-controller:latest
```
Comment (locriandev) [inline bundle/manifests/ingress-node-firewall.clusterserviceversion.yaml:249]: Can't this be updated to use `openshift` repo?

File: config/manager/kustomization.yaml [hunk 2]
```diff
@@ -9,6 +9,7 @@ patchesStrategicMerge:
 - manager_webhook_patch.yaml
 images:
 - name: controller
-  newName: quay.io/mmahmoud/controller
+  newName: quay.io/mmahmoud/ingress-node-firewall-controller
```
Comment (locriandev) [inline config/manager/kustomization.yaml:12]: Can't this be updated to use `openshift` repo?

File: bundle/manifests/ingress-node-firewall.clusterserviceversion.yaml (hunk 1 above)
Comment (msherif1234) [reply bundle/manifests/ingress-node-firewall.clusterserviceversion.yaml]: ofcourse I thought I did that not sure why it flipped back anyway I fixed both comments Thanks

**Pull request merged**